### ISSUE TITLE
Implement a 404 page

### ISSUE THREAD

**Issue by kantord** [kantord/LibreLingo#179]
It should also work with GitHub pages :smile_cat:

**Comment by Roshanjossey** [comment]
![7VE](https://user-images.githubusercontent.com/8488446/76566600-6253cb80-64ad-11ea-9bdd-0c45a8963e13.gif)

**Comment by navanchauhan** [comment]
I posted a request on open source designs for a mascot. After that I will create a simple 404 page with the mascot

**Comment by kantord** [comment]
thanks! I tried posting that request too, but their API was down or something :-)

**Comment by navanchauhan** [comment]
Now that we finally have a mascot, I'll be sending a PR in a couple of days

**Comment by navanchauhan** [comment]
LibreLingo is being hosted on GH Pages, right? According to GitHub Pages documentation, a simple 404.html in the root directory automatically becomes the 404 page without changing the code

**Comment by kantord** [comment]
Yes. The task is about creating the 404 page using Svelte

**Comment by navanchauhan** [comment]
![image](https://user-images.githubusercontent.com/29234112/80571001-76319d80-8a19-11ea-829d-c5bf90d924d4.png)
![image](https://user-images.githubusercontent.com/29234112/80571008-79c52480-8a19-11ea-9475-10f43c5a1fc8.png)

I have tried to keep the same design style as the index page. GitHub pages requires a 404.html placed at the root of the deployment, I think we should create a static `404.html` file and it'll be copied every time we run yarn export

Please tell me anything that should be changed, this svelte page can handle all errors but a static 404.html will only display 404.html

**Comment by navanchauhan** [comment]
@kantord can you look at this, I'll make a PR if this is fine

**Comment by kantord** [comment]
@navanchauhan we have a better fitting mascot too, can you please use that one :-) https://github.com/kantord/LibreLingo/blob/master/branding/LibreLingo_Mascot/GreyShadow/Turtle4.svg

Also, I would change the text a bit:

> Oops! 404 - Not found
> We couldn't find the page you were looking for

And then add a button that links back to the home page:

> Go to the home page

**Comment by kantord** [comment]
Otherwise, looks good to me!

**Comment by doamatto** [comment]
@navanchauhan checking if you're still working on this

**Comment by kantord** [comment]
@doamatto feel free to take over if you like: https://github.com/kantord/LibreLingo/pull/432#issuecomment-626219716

**Comment by doamatto** [comment]
Ah, gotcha. Regardless, this issue is blocked by #637/#618. Once some of [the issues](https://github.com/kantord/LibreLingo/pull/637#issuecomment-684036337) are resolved, I'll make a PR to keep compliant with the redesign.

**Comment by kantord** [comment]
@doamatto it doesn't have to be blocked by #637/#618. Since nobody is working full time on these things, PRs can move slowly, so in this case feel free to start working on the 404 page too, if you wish.

You can simply use the current designs, and we'll rebase the redesign PR. Or is there some pitfall that I'm not seeing?

**Comment by doamatto** [comment]
> Or is there some pitfall that I'm not seeing?
It's mostly a matter on making sure we don't have to use the old design and just go straight to the new one.

**Comment by navanchauhan** [comment]
@doamatto oof, I haven't had anytime to contribute to open-source projects for a while.. If you want to pick it up, I have no problem because I might be a bit out of touch with the code now.

I think if you want, you can try salvaging some code and create a new PR and I'll close this one?

**Comment by doamatto** [comment]
I'll try to do one up in the near future. I'm a bit slammed with school right now, though.
